app/models/creator/creator.ts

```diff
@@ -23,7 +23,7 @@ export const CreatorModel = UserModel
   .volatile(() => ({
     hasFetchedDetails: false,
     isFetchingDetails: false,
-    isFollowing: false,
+    isFollowing: undefined as boolean,
```

Comment:
Weird hack, when will `isFollowing` be `false` and when will it be `undefined`? Can we just add a new field for this?
Seems `isFollowing` now has extra functionality which it shouldn't have?